From bb4801a6d301cbc2cd60b60fe3a1c23580291796 Mon Sep 17 00:00:00 2001
From: Nicolas Ballesteros <nicolas.ballesteros@canonical.com>
Date: Thu, 15 Oct 2026 11:24:54 -0500
Subject: [PATCH] fix: bump System.Security.Cryptography.Pkcs/Xml to 10.0.13

Addresses GHSA-846f-f764-76mx, GHSA-2fmv-rv2c-59x7 (Pkcs) and
GHSA-7hqp-7c5m-4vq6, GHSA-c399-9853-6jmj, GHSA-r955-8q5j-3h83 (Xml).
Without this, NuGet audit fails restore with NU1902/NU1903 when using
nuget.org. See https://github.com/PowerShell/PowerShell/issues/28219
---
 src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj  | 4 ++--
 .../System.Management.Automation.csproj                       | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj b/src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj
index 3d503a47a..641ab4e6d 100644
--- a/src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj
+++ b/src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj
@@ -58,9 +58,9 @@
     <PackageReference Include="System.Net.Http.WinHttpHandler" Version="10.0.5" />
     <PackageReference Include="System.Reflection.Context" Version="10.0.5" />
     <PackageReference Include="System.Runtime.Caching" Version="10.0.5" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="10.0.10" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="10.0.13" />
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="10.0.5" />
-    <PackageReference Include="System.Security.Cryptography.Xml" Version="10.0.10" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="10.0.13" />
     <PackageReference Include="System.Security.Permissions" Version="10.0.5" />
     <!-- the following package(s) are from https://github.com/dotnet/wcf -->
     <PackageReference Include="System.ServiceModel.Http" Version="10.0.652802" />
diff --git a/src/System.Management.Automation/System.Management.Automation.csproj b/src/System.Management.Automation/System.Management.Automation.csproj
index 47354f62c..ffc0a9ba3 100644
--- a/src/System.Management.Automation/System.Management.Automation.csproj
+++ b/src/System.Management.Automation/System.Management.Automation.csproj
@@ -38,7 +38,7 @@
     <PackageReference Include="System.Diagnostics.EventLog" Version="10.0.5" />
     <PackageReference Include="System.DirectoryServices" Version="10.0.5" />
     <PackageReference Include="System.Management" Version="10.0.5" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="10.0.5" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="10.0.13" />
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="10.0.5" />
     <PackageReference Include="System.Security.Permissions" Version="10.0.5" />
     <!-- the following package(s) are from the powershell org -->
-- 
2.43.0

